File: C _Program/circle.c
//wap to accept radius of circle 
//print area and circumference of circle
#include<stdio.h>
void main()
{
	int radius;
	float area,circ, pi=3.14f;
	printf("enter radius");
	scanf("%d",&radius);
	area=pi*radius*radius;
	circ=2*pi*radius;
	printf("\n area of circle=%2f",area);
	printf("\n circ of circle =%2f",circ);
}

File: Java_Programs/javaexamples folder/Circle.java
//wap to accept radius of a circle and print area and circ of circle
import java.util.*;
class Circle
{
public static void main(String []abc)
{
int radius;
float area,circ,pi=3.14f;

Scanner s=new Scanner(System.in);
System.out.println("enter radius ");
area=pi*radius*radius;
circ=2*pi*radius;
System.out.println("area : " + area);
System.out.println("circ:" + circ);
}
}